hyperspy/tests/drawing/test_plot_signal2d.py

```diff
@@ -347,6 +347,28 @@ def test_plot_images_single_image(mpl_cleanup):
     ax = hs.plot.plot_images(image0, saturated_pixels=0.1)
     return ax[0].figure
 
+def test_plot_images_multi_signal_w_axes_replot(mpl_cleanup):
+    imdata = np.random.rand(3, 5, 5)
+    imgs = hs.signals.Signal2D(imdata)
+    img_list = [imgs, imgs.inav[:2], imgs.inav[0]]
+    subplots=hs.plot.plot_images(img_list, axes_replot=True)
```

Comment:
Just a detail to keep things tidy: adding `axes_decor=None` would avoid sending a warning to the logger.